internal/rbd/group.go

```diff
@@ -77,3 +77,7 @@ func (rv *rbdVolume) RemoveFromGroup(ctx context.Context, vg types.VolumeGroup)
 
 	return librbd.GroupImageRemove(ioctx, name, rv.ioctx, rv.RbdImageName)
 }
+
+func (rv *rbdVolume) ToMirror() types.Mirror {
```

Comment:
Thanks! Would it make sense to have some checking if mirror attributes are valid?

Reply:
Not for now, do you have any specific checks in your mind?

Reply:
Not really. Mostly just to have the function match `func (*rbdVolume) ToCSI(ctx) (csi.Volume, error)`

Reply:
the signature is matching now, it returns nil for now but can be updated later as required.